README.md

```diff
@@ -16,10 +16,34 @@ We strongly advise creating a new IAM user just for Route53 just in case.
 
 ### Retrieving currently live domain
 ```
-/bin/current_live_environment
+env PYTHONPATH=. /bin/current_live_environment
```

Comment:
Is this `fish` shell specific? Might be worth mentioning that in the README.

Reply:
```
sh-4.4$ env PYTHONPATH=. bin/rollback
Flipping live to test-blue.example.co.uk.
```

Reply:
@davidwinter ^